front/src/modules/settings/components/SettingsIconSection.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from '@emotion/styled';
+
+import { IconPicker } from '@/ui/input/components/IconPicker';
+import { H2Title } from '@/ui/typography/components/H2Title';
+
+import ArrowRight from '../assets/ArrowRight.svg';
+
+import { IconWithLabel } from './IconWithLabel';
+
+const StyledContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
```

Comment:
is this align-self property useful? I see you add it to different containers but I am not sure it is needed

Reply:
I will check

Reply:
It doesn't seem useful here, I removed it in every file I added. It was on the figma css. This property is also used at several other locations in the code, we should maybe check if those are useful too.

Reply:
Yes, I usually remove them when I refactor :p 
We should only introduce code purposefully!